rel/emqx_vars

```diff
@@ -7,11 +7,15 @@ REL_VSN="{{ release_version }}"
 ERTS_VSN="{{ erts_vsn }}"
 ERL_OPTS="{{ erl_opts }}"
 RUNNER_BIN_DIR="{{ runner_bin_dir }}"
-RUNNER_LOG_DIR="{{ runner_log_dir }}"
 RUNNER_LIB_DIR="{{ runner_lib_dir }}"
+IS_ELIXIR="${IS_ELIXIR:-{{ is_elixir }}}"
+
+if [ -z "$RUNNER_LOG_DIR" ]; then
```

Comment:
Add a comment like below to avoid future churns:
```
## Allow users to pre-set `RUNNER_LOG_DIR` because it only affects boot commands like `start` and `console`,
## but not other commands such as `ping` and `ctl`.
```

Reply:
DONE